types/src/front/api_handlers/internal/agent_configuration.ts

```diff
@@ -111,6 +111,50 @@ export const PostOrPatchAgentConfigurationRequestBodySchema = t.type({
               })
             ),
           }),
+          t.type({
```

Comment:
Can we move this to a dedicated codec/manifest that belongs to `process_configuration`?

Reply:
When we design that manifest yes